proxy/src/http/org.rs

```diff
@@ -168,7 +207,33 @@ impl ToDocumentedType for RegisterInput {
     }
 }
 
-#[allow(clippy::non_ascii_literal, clippy::result_unwrap_used)]
+/// Bundled input data for org list.
+#[derive(Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ListInput {
+    /// Id of the User.
+    id: String,
+}
+
+impl ToDocumentedType for ListInput {
+    fn document() -> document::DocumentedType {
+        let mut properties = std::collections::HashMap::with_capacity(1);
+        properties.insert(
+            "id".into(),
+            document::string()
+                .description("ID of the user")
+                .example("alice"),
+        );
+
+        document::DocumentedType::from(properties).description("Input for org list")
+    }
+}
+
+#[allow(
+    clippy::non_ascii_literal,
```

Comment:
This is only necessary if we have literal unicode like emojis in our test assertions.

Reply:
👍 Removed